modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'common/exceptions/record_not_found'
+require 'evss/letters/download_service'
+require 'evss/letters/service'
+
+module Mobile
+  module V0
+    class LettersController < ApplicationController
+      before_action { authorize :evss, :access? }
+
+      def index
+        service_response = EVSS::Letters::Service.new(@current_user).get_letters
+        response_template = OpenStruct.new
```

Comment:
OpenStruct takes a hash, so you could do
```ruby
response_template = OpenStruct.new(service_response.to_h)
response_template.id = @current_user.id
render json: Mobile::V0::LettersSerializer.new(response_template)
```

Reply:
A regular struct will always be faster though
``` ruby
LettersStruct = Struct.new(:id, :letters) # below the LettersController class end

service_response = service.get_letters
response_template = LettersStruct.new(@current_user.id, service_response.letters)
render json: Mobile::V0::LettersSerializer.new(response_template)
```

Reply:
And... since this is all to do with how the data is serialized you could bend the serializer to take an id
```ruby
module Mobile
  module V0
    class LettersSerializer
      include FastJsonapi::ObjectSerializer
      
      set_type :letters
      attributes :letters
      
      def initialize(id, resource, options = {})
        resource = LettersStruct.new(id, resource.letters)
        super(resource, options = {})
      end
    end
    
    LettersStruct = Struct.new(:id, :letters)
  end
end
```